src/Components/NewComment.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from "react";
+import { postAComment } from "../assets/Requests/api";
+
+const NewComment = ({ article_id, setComments }) => {
+  const [comment, setComment] = useState("");
+  const [username, setUsername] = useState("jessjelly");
+
+  useEffect(() => {
+    setUsername("jessjelly");
+  }, []);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    postAComment(article_id, username, comment).then((newComment) => {
+      setComments((currComments) => {
+        return [newComment, ...currComments];
+      });
+    });
```

Comment:
Would be a good idea to have a catch!

Reply:
Also needs to have 'something' to give the user feedback if there is an error, and a comment post is unsuccessful